pkg/vpc/importer.go

```diff
@@ -119,19 +119,18 @@ func (si *SpecConfigImporter) SharedNodeSecurityGroup() *gfnt.Value {
 	return si.ClusterSecurityGroup()
 }
 
-// SecurityGroups returns a gftn slice of the ControlPlaneSecurityGroup
-// and the ClusterSecurityGroup
+// SecurityGroups returns a gfnt slice of the ClusterSecurityGroup
 func (si *SpecConfigImporter) SecurityGroups() gfnt.Slice {
-	return gfnt.Slice{si.ControlPlaneSecurityGroup(), si.ClusterSecurityGroup()}
+	return gfnt.Slice{si.ClusterSecurityGroup()}
```

Comment:
This change only affects unowned clusters. For Owned clusters we are already doing this. I except this is actually not doing anything, but I have to further check the code.

Reply:
> This change only affects unowned clusters. For Owned clusters we are already doing this. I except this is actually not doing anything, but I have to further check the code.

why would this change only affect unowned? I'm slightly confused :/

Reply:
ahh makes sense, thank you!